aspnet/publishing/index.rst

```diff
@@ -13,3 +13,8 @@ Publishing and Deployment
     how-to-customize
     iis-with-msdeploy
     web-publishing-vs
+
+Related Resources
+-----------------
+
+* :doc:`/tutorials/publish-to-azure-webapp-using-vs`
```

Comment:
I think you can just include this topic in the TOC along side the other Azure publishing topics.